base/exports.jl

```diff
@@ -1204,6 +1204,7 @@ export
 # multiprocessing
     addprocs,
     asyncmap,
+    cluster_cookie,
```

Comment:
Only required by folks writing their own ClusterManagers. But I guess we could document its use as `Base. cluster_cookie()`